src/modules/userCreationRequestSchemas.ts

```diff
@@ -129,12 +130,22 @@ export const getCreateUserCreationRequestInternalEmployeeSchema = () =>
         workModeComment: z.string().optional(),
         extraEquipment: z.string().optional(),
         workSpace: z.string().optional(),
-        personalEmail: z.string().email(tr('Not a valid email')).optional().or(z.literal('')),
+        personalEmail: z
```

Comment:
Why are you use the `zod` validation? Why are you scaring user?
The easiest way is transforming the component  input value 

For example

```ts
const handleEmailChange = useCallback<React.ChangeEventHandler<HTMLInputElement>>((event) => setValue('email', event.target.value?.toLowerCase()), [setValue])
```